eng/Version-Increment.ps1

```diff
@@ -17,85 +17,128 @@ The Name of the Package
 .PARAMETER PackageDirName
 Used in the case where the package directory name is different from the package name. e.g in cognitiveservice packages
 
-.EXAMPLE
-Updating package version for core
+.PARAMETER NewVersionString
```

Comment:
minor but now that this also supports setting a version I'm not sure "Version-Increment" is the best name for the script. Perhaps having it called "Update-Version" with an `-increment` option which is mutually exclusive with the `-newVersion` option.